.editorconfig

```diff
@@ -6,7 +6,10 @@
 
 root = True
 
-[*.kt]
+[*.{kt,kts}]
 indent_size = 4
 indent_style = space
 
+ij_kotlin_allow_trailing_comma_on_call_site=true
+ij_kotlin_allow_trailing_comma=true
```

Comment:
Excuse lack of knowledge of configuring ktlint, but I was wondering if we actually needed to enable this manually in our own `.editorconfig` since I felt this was enabled by default upstream. https://github.com/pinterest/ktlint/pull/1583

Reply:
I just changed the update version from 0.46.1 to 0.47.0. I still haven't gotten to the part of dropping that commit )in 0.46.1 the rule was disabled by default)

Reply:
It seems that we do need to enable it, it is not imposed by default.

Reply:
Can we also squash the 2 commits while we are at? I think all the 2 commits are related and I don't really see any need to separate them. I think we're looking for a fine balance between micro-commits and meaningful context of the intended changes in a commit. I would think this from the perspective of someone looking at the git diff of just the ktlint version bump commit and believing we only need to bump the version number, but completely missing out on the context of also re-running and generating a baseline.

Reply:
The 2 commits are independent. You can upgrade without enabling this rule.

Reply:
I am okay with enabling a new rule being a separate commit, but would prefer the ktlint upgrade and baseline to be together in this case. The latter is more to demonstrate that upgrading ktlint requires some change for anyone looking at the diff.

Reply:
Feel free to add `needs landing (squashed)` label. IMO these commits are separate.